webapp_v2/src/components/common/TabWrapper.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, useMemo } from 'react';
+import { styled } from '@mui/material';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Box from '@mui/material/Box';
+import { Outlet, useLocation, useNavigate } from 'react-router-dom';
+import { StudyMetadata } from '../../common/types';
+
+export const StyledTab = styled(Tabs, { shouldForwardProp: (prop) => prop !== 'border' && prop !== 'tabStyle' })<{border?: boolean, tabStyle?: 'normal' | 'withoutBorder'}>(({ theme, border, tabStyle }) => ({
+  width: '98%',
+  ...(border === true && {
+    borderBottom: 1,
+    borderColor: 'divider',
+  }),
+  ...(tabStyle && tabStyle === 'withoutBorder' && {
+    '& .MuiTabs-indicator': {
+      display: 'none',
+    },
+  }),
+}));
+
+interface Props {
+    study: StudyMetadata | undefined,
+    tabList: Array<{ label: string, path: string }>;
+    border?: boolean;
+    tabStyle?: 'normal' | 'withoutBorder';
+}
+
+function BasicTabs(props: Props) {
+  const { study, tabList, border, tabStyle } = props;
+  const location = useLocation();
+  const navigate = useNavigate();
+  const [value, setValue] = React.useState(0);
+
+  useEffect(() => {
+    const getDefaultIndex = () : number => {
+      const index = tabList.findIndex((elm) => location.pathname.substring(0, elm.path.length) === elm.path);
+      if (index >= 0) return index;
+      return 0;
+    };
+    if (study) {
+      setValue(getDefaultIndex);
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [study, tabList]);
```

Comment:
just add the location into the dependencies (needed to fix the previous page system)